packages/discord.js/src/structures/ThreadChannel.js

```diff
@@ -352,7 +352,7 @@ class ThreadChannel extends BaseChannel {
   async edit(options) {
     const newData = await this.client.rest.patch(Routes.channel(this.id), {
       body: {
-        name: (options.name ?? this.name).trim(),
+        name: options.name?.trim(),
```

Comment:
The API performs trimming too, so you might as well remove that also, just like how it's done for channels:

https://github.com/discordjs/discord.js/blob/c484e3de250e2ed52f9651b1fc7f2b27990ef632/packages/discord.js/src/managers/GuildChannelManager.js#L317

Reply:
I wanted to remove it but I didn't dare XD I'll do it now 👍